examples/tutorials/CFD_Theory_Tutorial.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
Please add this on to the CSV files to make sure it gets rendered

Reply:
Check https://github.com/deepchem/deepchem.github.io/ for website rendering of tutorials to make sure it gets added